### ISSUE TITLE
Windows: unable to install binary CLI module due to permissions error in CLI

### ISSUE THREAD

**Issue by nickbp** [dcos/dcos-cli#741]
Please answer the following questions before submitting your issue. Thanks!

### What version of DC/OS + DC/OS CLI are you using (`dcos --version`)?
If DC/OS CLI version < 0.4.5, DC/OS version can be found at `http://dcos_url/dcos-metada/dcos-version.json`

C:\Users\nick\Downloads>dcos --version
dcoscli.version=0.4.10
dcos.version=1.8-dev
dcos.commit=87f744716be332be5347280508a4d197beeeedd6
dcos.bootstrap-id=20944ba7d8867a345b41ae50607a13b94b24f57f

### What operating system and version are you using?

Windows 8.1
### What did you do?
If possible please provide a recipe for reproducing.

install binary CLI module via 'dcos package install --cli kafka'

### What did you expect to see?

binary CLI module installs successfully

### What did you see instead?

```
C:\Users\nick\Downloads>dcos.exe
Command line utility for the Mesosphere Datacenter Operating
System (DC/OS). The Mesosphere DC/OS is a distributed operating
system built around Apache Mesos. This utility provides tools
for easy management of a DC/OS installation.

Available DC/OS commands:

        auth            Authenticate to DC/OS cluster
        config          Manage the DC/OS configuration file
        help            Display help information about DC/OS
        marathon        Deploy and manage applications to DC/OS
        node            Administer and manage DC/OS cluster nodes
        package         Install and manage DC/OS software packages
        service         Manage DC/OS services
        task            Manage DC/OS tasks

Get detailed command description with 'dcos <command> --help'.

C:\Users\nick\Downloads>dcos config set core.dcos_url https://gabriel-r-elasticl
-1kb545ofwxii2-442801080.us-west-2.elb.amazonaws.com/
[core.dcos_url]: set to 'https://gabriel-r-elasticl-1kb545ofwxii2-442801080.us-w
est-2.elb.amazonaws.com/'

C:\Users\nick\Downloads>dcos auth login
An SSL error occurred. To configure your SSL settings, please run: `dcos config
set core.ssl_verify <value>`
<value>: Whether to verify SSL certs for HTTPS or path to certs

C:\Users\nick\Downloads>dcos config set core.ssl_verify false
[core.ssl_verify]: set to 'false'

C:\Users\nick\Downloads>dcos auth login
gabriel-r-elasticl-1kb545ofwxii2-442801080.us-west-2.elb.amazonaws.com's usernam
e: bootstrapuser
bootstrapuser@gabriel-r-elasticl-1kb545ofwxii2-442801080.us-west-2.elb.amazonaws
.com's password:
Login successful!

C:\Users\nick\Downloads>dcos package install --cli kafka
Installing CLI subcommand for package [kafka] version [1.1.10-0.10.0.0]
Error installing 'kafka' package.
Run with `dcos --log-level=ERROR` to see the full output.

C:\Users\nick\Downloads>dcos package repo list
dev-universe: http://universe.marathon.mesos:8085/repo
Universe: https://universe.mesosphere.com/repo

C:\Users\nick\Downloads>dcos --log-level=DEBUG package install --cli kafka
MainThread: 2016-08-24 11:36:20,661 dcos\http.py:_request:87 - Sending HTTP ['ge
t'] to ['https://gabriel-r-elasticl-1kb545ofwxii2-442801080.us-west-2.elb.amazon
aws.com/capabilities']: {'Content-Type': 'application/vnd.dcos.capabilities+json
;charset=utf-8;version=v1', 'Accept': 'application/vnd.dcos.capabilities+json;ch
arset=utf-8;version=v1'}
MainThread: 2016-08-24 11:36:20,661 site-packages\requests\packages\urllib3\conn
ectionpool.py:_new_conn:758 - Starting new HTTPS connection (1): gabriel-r-elast
icl-1kb545ofwxii2-442801080.us-west-2.elb.amazonaws.com
MainThread: 2016-08-24 11:36:21,458 site-packages\requests\packages\urllib3\conn
ectionpool.py:_make_request:387 - "GET /capabilities HTTP/1.1" 401 None
MainThread: 2016-08-24 11:36:21,458 dcos\http.py:_request:117 - Received HTTP re
sponse [401]: {'Connection': 'keep-alive', 'Date': 'Wed, 24 Aug 2016 18:36:21 GM
T', 'Server': 'openresty/1.9.15.1', 'Content-Type': 'text/html; charset=UTF-8',
'Transfer-Encoding': 'chunked', 'WWW-Authenticate': 'acsjwt'}
MainThread: 2016-08-24 11:36:21,458 dcos\util.py:timer:529 - duration: dcos.http
._request: 0.80s
MainThread: 2016-08-24 11:36:21,458 dcos\http.py:_request:87 - Sending HTTP ['ge
t'] to ['https://gabriel-r-elasticl-1kb545ofwxii2-442801080.us-west-2.elb.amazon
aws.com/capabilities']: {'Content-Type': 'application/vnd.dcos.capabilities+json
;charset=utf-8;version=v1', 'Accept': 'application/vnd.dcos.capabilities+json;ch
arset=utf-8;version=v1'}
MainThread: 2016-08-24 11:36:21,474 site-packages\requests\packages\urllib3\conn
ectionpool.py:_new_conn:758 - Starting new HTTPS connection (1): gabriel-r-elast
icl-1kb545ofwxii2-442801080.us-west-2.elb.amazonaws.com
MainThread: 2016-08-24 11:36:21,599 site-packages\requests\packages\urllib3\conn
ectionpool.py:_make_request:387 - "GET /capabilities HTTP/1.1" 200 103
MainThread: 2016-08-24 11:36:21,599 dcos\http.py:_request:117 - Received HTTP re
sponse [200]: {'Server': 'openresty/1.9.15.1', 'Content-Length': '103', 'Content
-Type': 'application/vnd.dcos.capabilities+json;charset=utf-8;version=v1', 'Date
': 'Wed, 24 Aug 2016 18:36:22 GMT', 'Connection': 'keep-alive'}
MainThread: 2016-08-24 11:36:21,599 dcos\util.py:timer:529 - duration: dcos.http
._request: 0.14s
MainThread: 2016-08-24 11:36:21,615 dcos\http.py:_request:87 - Sending HTTP ['po
st'] to ['https://gabriel-r-elasticl-1kb545ofwxii2-442801080.us-west-2.elb.amazo
naws.com/package/describe']: {'Content-Type': 'application/vnd.dcos.package.desc
ribe-request+json;charset=utf-8;version=v1', 'Accept': 'application/vnd.dcos.pac
kage.describe-response+json;charset=utf-8;version=v2'}
MainThread: 2016-08-24 11:36:21,615 site-packages\requests\packages\urllib3\conn
ectionpool.py:_new_conn:758 - Starting new HTTPS connection (1): gabriel-r-elast
icl-1kb545ofwxii2-442801080.us-west-2.elb.amazonaws.com
MainThread: 2016-08-24 11:36:21,755 site-packages\requests\packages\urllib3\conn
ectionpool.py:_make_request:387 - "POST /package/describe HTTP/1.1" 401 None
MainThread: 2016-08-24 11:36:21,755 dcos\http.py:_request:117 - Received HTTP re
sponse [401]: {'Connection': 'keep-alive', 'Date': 'Wed, 24 Aug 2016 18:36:22 GM
T', 'Server': 'openresty/1.9.15.1', 'Content-Type': 'text/html; charset=UTF-8',
'Transfer-Encoding': 'chunked', 'WWW-Authenticate': 'acsjwt'}
MainThread: 2016-08-24 11:36:21,755 dcos\util.py:timer:529 - duration: dcos.http
._request: 0.14s
MainThread: 2016-08-24 11:36:21,755 dcos\http.py:_request:87 - Sending HTTP ['po
st'] to ['https://gabriel-r-elasticl-1kb545ofwxii2-442801080.us-west-2.elb.amazo
naws.com/package/describe']: {'Content-Type': 'application/vnd.dcos.package.desc
ribe-request+json;charset=utf-8;version=v1', 'Accept': 'application/vnd.dcos.pac
kage.describe-response+json;charset=utf-8;version=v2'}
MainThread: 2016-08-24 11:36:21,755 site-packages\requests\packages\urllib3\conn
ectionpool.py:_new_conn:758 - Starting new HTTPS connection (1): gabriel-r-elast
icl-1kb545ofwxii2-442801080.us-west-2.elb.amazonaws.com
MainThread: 2016-08-24 11:36:22,037 site-packages\requests\packages\urllib3\conn
ectionpool.py:_make_request:387 - "POST /package/describe HTTP/1.1" 200 39908
MainThread: 2016-08-24 11:36:22,068 dcos\http.py:_request:117 - Received HTTP re
sponse [200]: {'Server': 'openresty/1.9.15.1', 'Content-Length': '39908', 'Conte
nt-Type': 'application/vnd.dcos.package.describe-response+json;charset=utf-8;ver
sion=v2', 'Date': 'Wed, 24 Aug 2016 18:36:22 GMT', 'Connection': 'keep-alive'}
MainThread: 2016-08-24 11:36:22,068 dcos\util.py:timer:529 - duration: dcos.http
._request: 0.31s
Installing CLI subcommand for package [kafka] version [1.1.10-0.10.0.0]
MainThread: 2016-08-24 11:36:22,083 dcos\util.py:ensure_dir_exists:129 - Creatin
g directory: 'C:\\Users\\nick\\.dcos\\subcommands\\kafka'
MainThread: 2016-08-24 11:36:22,083 site-packages\requests\packages\urllib3\conn
ectionpool.py:_new_conn:758 - Starting new HTTPS connection (1): downloads.mesos
phere.com
MainThread: 2016-08-24 11:36:22,177 site-packages\requests\packages\urllib3\conn
ectionpool.py:_make_request:387 - "GET /kafka/assets/1.1.10-0.10.0.0/dcos-kafka.
exe HTTP/1.1" 200 8566784
MainThread: 2016-08-24 11:36:22,943 dcos\util.py:ensure_dir_exists:129 - Creatin
g directory: 'C:\\Users\\nick\\.dcos\\subcommands\\kafka\\env\\Scripts'
MainThread: 2016-08-24 11:36:23,021 dcos\subcommand.py:_install_with_binary:492
- [WinError 32] The process cannot access the file because it is being used by a
nother process: 'C:\\Users\\nick\\AppData\\Local\\Temp\\tmpepdgtcl_'
Traceback (most recent call last):
  File "shutil.py", line 523, in move
PermissionError: [WinError 32] The process cannot access the file because it is
being used by another process: 'C:\\Users\\nick\\AppData\\Local\\Temp\\tmpepdgtc
l_' -> 'C:\\Users\\nick\\.dcos\\subcommands\\kafka\\env\\Scripts\\dcos-kafka'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "dcos\subcommand.py", line 464, in _install_with_binary
  File "shutil.py", line 536, in move
PermissionError: [WinError 32] The process cannot access the file because it is
being used by another process: 'C:\\Users\\nick\\AppData\\Local\\Temp\\tmpepdgtc
l_'
Error installing 'kafka' package.
Run with `dcos --log-level=ERROR` to see the full output.

C:\Users\nick\Downloads>
```

**Comment by tamarrow** [comment]
This is possible due to Windows not letting you move a file that is open. Will verify and see if copying the file first fixes the issue.

**Comment by tamarrow** [comment]
fixed with #742
